Implementation/Composing/index.md

```diff
@@ -154,10 +153,10 @@ Ordering of composers is important, the last one added can override a previously
 It's an implementation of IUserComposer, that provides a quicker way to add a custom Component to the Component's collection. Creating a C# class that inherits from ComponentComposer&lt;YourComponentType&gt; will automatically add YourComponentType to the collection of Components. In the example above, the SubscribeToContentServiceSavingComposer for the SubscribeToContentServiceSavingComponent could have been written more conveniently as:
```

Comment:
ComponentComposer is not an implementation of IUserComposer, it's an implementation of IComposer